exercises/simple-cipher/simple-cipher.spec.js

```diff
@@ -24,21 +24,22 @@ describe('Random key cipher', function () {
   });
 });
 
+/* eslint-disable no-new */
 describe('Incorrect key cipher', function () {
   xit('throws an error with an all caps key', function () {
-    expect( function () {
+    expect(function () {
       new Cipher('ABCDEF');
     }).toThrow(new Error('Bad key'));
   });
 
   xit('throws an error with a numeric key', function () {
-    expect( function () {
+    expect(function () {
       new Cipher('12345');
     }).toThrow(new Error('Bad key'));
   });
 
   xit('throws an error with an empty key', function () {
-    expect( function () {
+    expect(function () {
       new Cipher('');
     }).toThrow(new Error('Bad key'));
   });
```

Comment:
Be sure to add the complementary `enable` line for the eslint rule.

Reply:
No problem.

You need to define the _end_ of the `disable` block, otherwise the entire rest of the file will be ignored for that ESlint rule.  EG:


```javascript
  /* eslint-disable no-new */
describe('Incorrect key cipher', function () {
...
});
  /* eslint-enable no-new */
```

Notice that this block begins with `disable` and ends with `enable`, which turns that linting rule back on.

Reply:
Thanks for explaining! Just pushed the change. Final issue woohoo!